*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 19:24:49 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1fec0c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flask.py", line 1, in <module>
    from flask import Flask
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flask.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
    from flask import Flask
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 34988)
spawned uWSGI worker 1 (pid: 34993, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 34993|app: -1|req: -1/1] 172.32.100.1 () {50 vars in 1105 bytes} [Sun Oct 23 19:58:31 2016] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
--- no python application found, check your startup logs for errors ---
[pid: 34993|app: -1|req: -1/2] 172.32.100.1 () {50 vars in 1025 bytes} [Sun Oct 23 19:58:31 2016] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Sun Oct 23 20:01:34 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 34993)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:01:35 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x193bf50
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 34988)
spawned uWSGI worker 1 (pid: 35810, cores: 1)
Sun Oct 23 20:01:35 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 35810)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:01:36 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x8e9f50
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 34988)
spawned uWSGI worker 1 (pid: 35815, cores: 1)
Sun Oct 23 20:01:56 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:02:00 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1dce0c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 35833)
spawned uWSGI worker 1 (pid: 35838, cores: 1)
Sun Oct 23 20:05:31 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 35838)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:05:32 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1a49f20
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "flask.uwsgi", line 1, in <module>
    from flasker import app as application 
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 35833)
spawned uWSGI worker 1 (pid: 35992, cores: 1)
Sun Oct 23 20:05:32 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:05:36 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x87c090
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "flask.uwsgi", line 1, in <module>
    from flasker import app as application 
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 35996)
spawned uWSGI worker 1 (pid: 36001, cores: 1)
Sun Oct 23 20:06:03 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36001)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:06:04 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1dbef50
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 35996)
spawned uWSGI worker 1 (pid: 36012, cores: 1)
Sun Oct 23 20:06:04 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36012)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:06:05 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x104af50
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 35996)
spawned uWSGI worker 1 (pid: 36017, cores: 1)
Sun Oct 23 20:06:07 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:06:11 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x8290c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 36021)
spawned uWSGI worker 1 (pid: 36026, cores: 1)
Sun Oct 23 20:06:19 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:06:23 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7bc0c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
Traceback (most recent call last):
  File "./flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
Traceback (most recent call last):
  File "flasker.py", line 1, in <module>
    from flask import Flask
  File "./flask.py", line 1, in <module>
ImportError: cannot import name Flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 36043)
spawned uWSGI worker 1 (pid: 36048, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 36048|app: -1|req: -1/1] 172.32.100.1 () {48 vars in 1047 bytes} [Sun Oct 23 20:07:25 2016] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
--- no python application found, check your startup logs for errors ---
[pid: 36048|app: -1|req: -1/2] 172.32.100.1 () {50 vars in 1025 bytes} [Sun Oct 23 20:07:25 2016] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Sun Oct 23 20:07:37 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:07:41 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x20260c0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
ImportError: No module named py
unable to load app 0 (mountpoint='') (callable not found or import error)
unable to find "application" callable in file flasker.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36108, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 36108|app: -1|req: -1/1] 172.32.100.1 () {48 vars in 1047 bytes} [Sun Oct 23 20:07:50 2016] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
--- no python application found, check your startup logs for errors ---
[pid: 36108|app: -1|req: -1/2] 172.32.100.1 () {50 vars in 1025 bytes} [Sun Oct 23 20:07:51 2016] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Sun Oct 23 20:12:42 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36108)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:12:43 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2300f20
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
ImportError: No module named py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36233, cores: 1)
Sun Oct 23 20:13:25 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36233)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:13:26 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x24c9f50
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
ImportError: No module named py
unable to load app 0 (mountpoint='') (callable not found or import error)
failed to open python file flask.uwsgi
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36246, cores: 1)
Sun Oct 23 20:13:41 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36246)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:13:42 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x26b7f20
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
failed to open python file flask.uwsgi
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36247, cores: 1)
Sun Oct 23 20:13:48 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36247)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:13:49 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x16fbf20
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x16fbf20 pid: 36098 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36252, cores: 1)
[pid: 36252|app: 0|req: 1/1] 172.32.100.1 () {48 vars in 1047 bytes} [Sun Oct 23 20:14:00 2016] GET / => generated 45 bytes in 3 msecs (HTTP/1.1 200) 2 headers in 79 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 36252|app: 0|req: 2/2] 172.32.100.1 () {50 vars in 1025 bytes} [Sun Oct 23 20:14:00 2016] GET /favicon.ico => generated 233 bytes in 19 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Sun Oct 23 20:18:54 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36252)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:18:55 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2298f20
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
ImportError: No module named py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36301, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 36301|app: -1|req: -1/1] 172.32.100.1 () {48 vars in 1000 bytes} [Sun Oct 23 20:19:04 2016] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
Sun Oct 23 20:20:10 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36301)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 7) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:20:11 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2595ea0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x2595ea0 pid: 36098 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36098)
spawned uWSGI worker 1 (pid: 36323, cores: 1)
[pid: 36323|app: 0|req: 1/1] 172.32.100.1 () {48 vars in 1000 bytes} [Sun Oct 23 20:20:18 2016] GET / => generated 45 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 79 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
Sun Oct 23 20:21:14 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:42:24 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1b85020
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x1b85020 pid: 36718 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 36718)
spawned uWSGI worker 1 (pid: 36723, cores: 1)
Sun Oct 23 20:43:16 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36723)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 6) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:43:17 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2397eb0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
ImportError: No module named py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36718)
spawned uWSGI worker 1 (pid: 36730, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 36730|app: -1|req: -1/1] 172.32.100.1 () {48 vars in 1000 bytes} [Sun Oct 23 20:43:23 2016] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
Sun Oct 23 20:43:38 2016 - received message 1 from emperor
...gracefully killing workers...
Gracefully killing worker 1 (pid: 36730)...
worker 1 buried after 1 seconds
binary reloading uWSGI...
chdir() to /usr/local/uwsgi/etc/uwsgi.d
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/usr/local/uwsgi/var/run/flask.sock)
running /usr/local/uwsgi/bin/uwsgi
*** has_emperor mode detected (fd: 6) ***
[uWSGI] getting INI configuration from flask.ini
*** Starting uWSGI 2.0.14 (64bit) on [Sun Oct 23 20:43:39 2016] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-4) on 19 October 2016 23:17:07
os: Linux-3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 2
current working directory: /usr/local/uwsgi/etc/uwsgi.d
detected binary path: /usr/local/uwsgi/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /opt/project/flask
your processes number limit is 3794
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /usr/local/uwsgi/var/run/flask.sock fd 3
Python version: 2.7.5 (default, Sep 15 2016, 22:37:39)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xeade90
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xeade90 pid: 36718 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 36718)
spawned uWSGI worker 1 (pid: 36743, cores: 1)
[pid: 36743|app: 0|req: 1/1] 172.32.100.1 () {48 vars in 1000 bytes} [Sun Oct 23 20:43:40 2016] GET / => generated 45 bytes in 2 msecs (HTTP/1.1 200) 2 headers in 79 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
Sun Oct 23 20:44:54 2016 - received message 0 from emperor
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
